model/SyncLog/Storage/RdsSyncLogStorage.php

```diff
@@ -0,0 +1,237 @@
+ use oat\oatbox\extension\script\MissingOptionException;
+ use oat\oatbox\service\ConfigurableService;
+ use oat\taoSync\model\SyncLog\SyncLogEntity;
+ use oat\taoSync\model\SyncLog\SyncLogFilter;
+ use common_exception_NotFound;
+ use common_persistence_SqlPersistence;
+ use Doctrine\DBAL\Connection;
+
+ class RdsSyncLogStorage extends ConfigurableService implements SyncLogStorageInterface
+ {
+     const SERVICE_ID = 'taoSync/RdsSyncLogStorage';
+
+     const OPTION_PERSISTENCE_ID = 'persistenceId';
+
+     const TABLE_NAME = 'synchronisation_log';
+
+     /**
+      * @var string
+      */
+     private $persistenceId = null;
+
+     /**
+      * @var common_persistence_SqlPersistence
+      */
+     private $persistence = null;
+
+     /**
+      * SyncLogService constructor.
+      * @param array $options
+      * @throws MissingOptionException
+      */
+     public function __construct(array $options)
+     {
+         parent::__construct($options);
+
+         if (!$this->hasOption(self::OPTION_PERSISTENCE_ID)) {
+             throw new MissingOptionException('Persistence ID is required option.', self::OPTION_PERSISTENCE_ID);
+         }
+         $this->persistenceId = $this->getOption(self::OPTION_PERSISTENCE_ID);
+     }
+
+     /**
+      * @return common_persistence_SqlPersistence
+      */
+     public function getPersistence()
+     {
+         if ($this->persistence === null) {
+             $this->persistence = $this->getServiceLocator()
+                 ->get(\common_persistence_Manager::SERVICE_ID)
+                 ->getPersistenceById($this->persistenceId);
+         }
+
+         return $this->persistence;
+     }
+
+     /**
+      * @return QueryBuilder
+      */
+     private function getQueryBuilder()
+     {
+         return $this->getPersistence()->getPlatForm()->getQueryBuilder()->from(self::TABLE_NAME);
+     }
+
+     /**
+      * Store SyncLogEntity in rds storage.
+      *
+      * @param SyncLogEntity $entity
+      * @return integer
+      */
+     public function create(SyncLogEntity $entity)
+     {
+         return $this->getPersistence()->insert(
+             self::TABLE_NAME,
+             [
+                 SyncLogStorageInterface::COLUMN_SYNC_ID => $entity->getSyncId(),
+                 SyncLogStorageInterface::COLUMN_BOX_ID => $entity->getBoxId(),
+                 SyncLogStorageInterface::COLUMN_ORGANIZATION_ID => $entity->getOrganizationId(),
+                 SyncLogStorageInterface::COLUMN_DATA => json_encode($entity->getData()),
+                 SyncLogStorageInterface::COLUMN_STATUS => $entity->getStatus(),
+                 SyncLogStorageInterface::COLUMN_REPORT => json_encode($entity->getReport()),
+                 SyncLogStorageInterface::COLUMN_STARTED_AT => $entity->getStartTime()->format(SyncLogEntity::DATE_TIME_FORMAT)
+             ]
+         );
+     }
+
+     /**
+      * Update synchronization log record.
+      *
+      * @param SyncLogEntity $entity
+      * @return mixed
```

Comment:
it would be better to return boolean value

Reply:
Fixed annotations to return integer (number of updated records)